packages/flutter/lib/src/material/stepper.dart

```diff
@@ -232,6 +240,27 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// give background color to header
+  final Color? backgroundHeaderColor;
+
+  /// set the BoderRadius to header
+  final BorderRadiusGeometry? headerRadius;
+
+  /// set the color to text of active step
+  final Color? activeCircleTextColor;
+
+  /// set the background color to step
+  final Color? circleColor;
+
+  /// set the underline color
+  final Color? lineColor;
+
+  /// set the color of the text to inactive step
+  final Color? inactiveTextColor;
+
+  /// set the color to check icon
+  final Color? iconCheckColor;
```

Comment:
It is slightly redundant that this only affects inactive steps, as that is only time a check is drawn; but I would still recommend we spell that out in this docstring. It would also be nice to add `StepperState` to the "See Also" section you see sprinkled across other docstrings.